src/test/java/org/fundacionjala/coding/Marcan/TestValidationEAN.java

```diff
@@ -0,0 +1,31 @@
+package org.fundacionjala.coding.Marcan;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by MarcandeaCabrera on 5/26/2017.
+ */
+public class TestValidationEAN {
+    /**
+     * This is a Test for verify if is true the result (equal to ultimate number).
+     */
+    @Test
+    public void firstTestForAssertTrueOfString() {
+        ValidationEAN stringEAN = new ValidationEAN();
+        assertTrue(stringEAN.verifyStringEAN("4003301018398"));
+    }
+
+    /**
+     * This is a Test for verify if is false the result (equal to ultimate number).
+     */
+    @Test
+    public void secondTestForAssertFalseOfString() {
+        ValidationEAN stringEAN = new ValidationEAN();
```

Comment:
You are repeating this code in all your tests
Please put them in @before method or convert it to a static method.